guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -8,6 +8,11 @@ All subscription information is available in your Red Hat Customer Portal accoun
 
 Before you can complete the tasks in this chapter, you must create a Red{nbsp}Hat Subscription Manifest in the Customer Portal.
 
+ifdef::satellite[]
```

Comment:
I wonder if the `ifdef` is really needed. Katello consumes subscriptions from the Red Hat portal the same way Satellite does (as it's the same codebase), and if the portal ever drops non-SCA management, Katello won't be able to support it either.

(Can't speak for Orcharino and how they manage this, but I would guess "same"?)